contracts/facets/GBMFacet.sol

```diff
@@ -247,25 +254,36 @@ contract GBMFacet is IGBM, IERC1155TokenReceiver, IERC721TokenReceiver, Modifier
         emit Auction_Initialized(_aid, id, amount, ca, tokenKind, _auctionPresetID);
         emit Auction_StartTimeUpdated(_aid, getAuctionStartTime(_aid), getAuctionEndTime(_aid));
         s.auctionNonce++;
+
+        //In order to start an auction with a minium starting price, you need to prepay the fees
+        if (_info.startingBid != 0) {
+            //Transfer the money of the seller to the GBM Diamond
+            uint256 prepaidFee = (_info.startingBid * 40) / 1000; //4% fee, hardcoded
+            IERC20(s.GHST).transferFrom(msg.sender, address(this), prepaidFee);
+
+            //Presettle the fee
+            uint256 _rem = _settleFees(_info.startingBid);
```

Comment:
Why are we using settleFees and not setleFeesWithRoyalty here? @orionstardust

Reply:
@cinnabarhorse 
starting bid is a fee. this is why we use `_settleFees`

Reply:
Got it